tests/unit/cag/test_fixed_increments.py

```diff
@@ -0,0 +1,382 @@
+        assert instance.table_name is None
+        assert instance._fixed_increments_column_name == f'{column_name}#increment'
+        assert instance._dtype is None
+        assert instance.increment_value == 10
+
+    def test__init__table_name(self):
+        """Test init sets attributes with table_name defined"""
+        # Setup
+        instance = FixedIncrements(column_name='a', increment_value=10, table_name='table1')
+
+        # Asserts
+        assert instance.column_name == 'a'
+        assert instance.increment_value == 10
+        assert instance._dtype is None
+        assert instance.table_name == 'table1'
+
+    @patch('sdv.cag.fixed_increments._validate_table_and_column_names')
+    def test__validate_pattern_with_metadata(
+        self,
+        _validate_table_and_column_names_mock,
+    ):
+        """Test validating the pattern with metadata."""
+        # Setup
+        instance = FixedIncrements(column_name='a', increment_value=5)
+        metadata = Metadata.load_from_dict({
+            'tables': {
+                'table': {
+                    'columns': {
+                        'id': {'sdtype': 'id'},
+                        'a': {'sdtype': 'numerical'},
+                    },
+                }
+            }
+        })
+
+        # Run
+        instance._validate_pattern_with_metadata(metadata)
+
+        # Assert
+        _validate_table_and_column_names_mock.assert_called_once()
+
+    def test__validate_pattern_with_metadata_incorrect_sdtype(self):
+        """Test it when the sdtype is not numerical."""
+        # Setup
+        instance = FixedIncrements(column_name='a', increment_value=2)
+        metadata = Metadata.load_from_dict({
+            'tables': {
+                'table': {
+                    'columns': {
+                        'id': {'sdtype': 'id'},
+                        'a': {'sdtype': 'datetime'},
+                    },
+                }
+            }
+        })
+        err_msg = re.escape(
+            "Column 'a' has an incompatible sdtype ('datetime')."
+            "The column sdtype must be 'numerical'."
+        )
+        # Run and Assert
+        with pytest.raises(PatternNotMetError, match=err_msg):
+            instance._validate_pattern_with_metadata(metadata)
+
+    def test__validate_pattern_with_metadata_invalid_column_name(self):
+        """Test it when the column name is not in metadata."""
+        # Setup
+        column_name = 'column'
+        instance = FixedIncrements(column_name=column_name, increment_value=2)
+        metadata = Metadata.load_from_dict({
+            'tables': {
+                'table': {
+                    'columns': {
+                        'id': {'sdtype': 'id'},
+                        'a': {'sdtype': 'datetime'},
+                    },
+                }
+            }
+        })
+        err_msg = re.escape(f"Table 'table' is missing columns '{column_name}")
+        # Run and Assert
```

Comment:
Missing new line for this and tests below

Reply:
Fixed here and in other spots:
https://github.com/sdv-dev/SDV/pull/2409/commits/ab7486461cdacaa4a32ab55016ba3204f780c406#diff-813b0ae6f76231c82778f818e85f968f48747005a684ea95e0abc9ba1c524b07R129